packages/plugin-core/src/WSUtilsV2.ts

```diff
@@ -21,6 +24,23 @@ export class WSUtilsV2 implements IWSUtilsV2 {
     this.extension = extension;
   }
 
+  static instance() {
```

Comment:
where is this one required? Can you mark this one with a comment saying all new implementations should avoid using this static reference?

Reply:
Added comment.